core/templates/domain/blog/blog-post.model.ts

```diff
@@ -30,11 +30,7 @@ export interface BlogPostBackendDict {
   'published_on'?: string;
 }
 export class BlogPostData {
-  // TODO(#13637): Remove the use of interstitial blog post
-  // The 'id' and 'thumbnailFilename' is 'null' for an interstitial
-  // blog post that is displayed in the editor until the actual
-  // is fetched from the backend.
-  _id: string | null;
+  _id: string;
```

Comment:
Should we also then update the `_thumbnailFilename` to be non-null?

Reply:
'id' cannot be null for an already existing blog post (draft or published), however for draft blog posts adding a thumbnail is not necessary and thus can be null!